src/xsar/sentinel1_dataset.py

```diff
@@ -420,12 +420,12 @@ def len_xtrack_m(self):
     @property
     def pixel_atrack_m(self):
         """atrack pixel spacing, in meters (relative to dataset)"""
-        return self.len_atrack_m / self.dataset.atrack.size
+        return self.s1meta.pixel_atrack_m * np.unique(np.round(np.diff(self._dataset['atrack'].values), 1))[0]
```

Comment:
@agrouaze , can you explain what you mean with "Do not forget that pixel_atrack_m is currently not aligned with s1meta.image['ground_pixel_spacing']" ?